app/controllers/admin/collections_controller.rb

```diff
@@ -143,7 +143,7 @@ def update
       ["group", "class", "user", "ipaddress"].each do |title|
         if params["submit_add_#{title}"].present?
           if params["add_#{title}"].present?
-            val = params["add_#{title}"].strip.remove_zero_width_chars
+            val = params["add_#{title}"]
```

Comment:
I think we may still need the `.strip` unless if we want to move that into the ApplicationController before_action as well, but I think that could wait for another PR.